### PR TITLE
 feat: Updated antispam and client files with code to remove any link…

### LINKED ISSUE #7: Deleting Promotional links posted by members
I propose this feature to stop new joining members from posting any links as they can be potential spam or NSFW content.
We can allow members who have a certain tag that they earn after being active for a while of given to them by a mod that will allow them to post a link.
Until then if the new members wish to share anything they can approach a "mod" or "Member"  to share anything that can be useful.
But We can let people post links that starts with https://github.com/

### PR DESCRIPTION
**Pull request by anushkrishnav**
Partial fix #7 
For now, it will delete all the links in the moderator-only and welcome  channel
I will be adding better filters to allow Github links and channel-specific permissions

### CODE REVIEW

File: src/modules/antispam.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import random
```
Comment (ricardoprins) [inline src/modules/antispam.py:1]: This import was never used

File: src/modules/antispam.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+from re import search
+links_not_allowed = (829038892145311775,829039782260244561) #moderator only and welcome channel
+
+def has_links(message):
+    url_regex = r"(?i)\b((?:https?://|www\d{0,3}[.]|[a-z0-9.\-]+[.][a-z]{2,4}/)(?:[^\s()<>]+|\(([^\s()<>]+|(\([^\s()<>]+\)))*\))+(?:\(([^\s()<>]+|(\([^\s()<>]+\)))*\)|[^\s`!()\[\]{};:'#\".,<>?«»“”‘’]))"
+    if search(url_regex, message.content) and if message.channel.id in links_not_allowed:
```
Comment (ricardoprins) [inline src/modules/antispam.py:7]: The second 'if' can be removed:

`if search(url_regex, message.content) and message.channel.id in links_not_allowed`

**Review by ricardoprins — CHANGES_REQUESTED** [review]
Comments added below :)

File: src/modules/antispam.py (hunk 2 above)
Comment (anushkrishnav) [reply src/modules/antispam.py:7]: I noticed it pushing 1 more commit it will be fixed

Comment (ricardoprins) [reply src/modules/antispam.py:7]: you can push it and then squash/rebase your commits into a single one - they'll be merged in this PR

**Pull request merged**